src/vector/index.js

```diff
@@ -23,22 +23,29 @@ sdk.loadModule(require('../modules/VectorConferenceHandler'));
 
 var lastLocationHashSet = null;
 
+
+function parseQueryParams(location) {
```

Comment:
AFAIK vector will be passed through browserify which supports core node moules so you can just do:
```javascript
var qs = require("querystring");
var params = qs.parse("foo=bar&baz=3");
console.log(params.foo) // "bar"
```

Using this is preferable because it handles common edge cases which are easy to forget when rolling your own (e.g. `foo=bar&baz` is valid (`baz` is a 0-length string) but this implementation will not catch that).